test/MerkleLib.utils.ts

```diff
@@ -120,10 +120,10 @@ export async function constructSingleChainTree(token: string, scalingSize = 1, r
   return { tokensSendToL2, realizedLpFees, leaves, tree };
 }
 
-export async function buildSlowRelayTree(relays: RelayData[]) {
```

Comment:
Did you just rename `RelayData` to `SlowFill` for better readability?

Reply:
No, it used to be that RelayData and SlowFill were the same data struct. That's no longer true. We have the payoutAdjustment parameter that is _not_ in the normal relayData struct (which is what we hash to identify relays).

Reply:
Ah i see